tgui/packages/tgui/interfaces/common/ReagentInfo.js

```diff
@@ -1,5 +1,15 @@
 import { Box, ColorBox, Flex, Icon, NoticeBox, Section, Tooltip } from '../../components';
 
+export const NoContainer = {
```

Comment:
Not going to require TypeScript here, but I would encourage you to look into it for the sake of making avoiding type errors easier for this type of structure.